frontend/src/app/(navfooter)/preset/page.tsx

```diff
@@ -0,0 +1,51 @@
+"use client"
+
+import React, { useState } from "react"
```

Comment:
Why are you importing `React`? This prevents tree-shaking
There's no need to type things `React.JSX.Element`; see the rest of the codebase

Reply:
I changed to `import { type JSX } from 'react'` when I could, otherwise removed it